src/common/common/obd_config.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <iostream>
+#include <unordered_map>
+
+std::unordered_map<std::string, std::string> parseObdData(const std::string& obdData);
+std::string serializeObdData(const std::unordered_map<std::string, std::string>& obdData);
```

Comment:
Seeing as these will only be used by the ObdConfig internally, these function can be defined in an anonymous namespace in obd_config.cpp rather than globally available